documentation/modules/con-brokered-infra-config.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// assembly-plans.adoc
+
+[id='con-brokered-infra-config-{context}']
+= Brokered infrastructure configuration
+
+`BrokeredInfraConfig` resources is used to configure infrastructure deployed by `brokered` address
+spaces. The brokered infra configuration is referenced by address space plans using a `defined-by`
+annotation. For more information, see xref:con-address-space-plans-{context}[].
+
+[source,yaml,options="nowrap"]
+.link:resources/brokered-infra-config-example.yaml[brokered-infra-config-example.yaml]
+----
+include::../common/brokered-infra-config-example.yaml[]
+----
+
+The `version` field represents the {ProductName} version used. When upgrading, {ProductName} will
```

Comment:
Change "represents" to "specifies"

"When upgrading, {ProductName} uses this field to determine whether to upgrade the infrastructure to the requested version." (?Did I capture the meaning correctly?)